docs/content/ingestion/flatten-json.md

```diff
@@ -68,37 +68,37 @@ To flatten this JSON, the parseSpec could be defined as follows:
         "expr": "foo.bar"
       },
       {
-        "type": "nested",
-        "name": "nested-metric",
+        "type": "path",
+        "name": "path-metric",
         "expr": "$.nestmet.val"
       },
       {
-        "type": "nested",
+        "type": "path",
         "name": "hello-0",
         "expr": "$.hello[0]"
       },
       {
-        "type": "nested",
+        "type": "path",
         "name": "hello-4",
         "expr": "$.hello[4]"
       },
       {
-        "type": "nested",
+        "type": "path",
         "name": "world-hey",
         "expr": "$.world.hey"
```

Comment:
While you're at it: this line and the other one with `world` below are incorrect. The correct syntax is `$.world[*].hey`.